src/redux/books/books.js

```diff
@@ -0,0 +1,23 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] With reference to the _duck pattern_, assigning values to _ADD_BOOK_ and _REMOVE_BOOK_ action types should be similar to:
```javascript
       const ADD_BOOK = bookstore/books/ADD_BOOK;
```

and

```javascript
      const REMOVE_BOOK = bookstore/books/REMOVE_BOOK;
```

Kindly ensure to improve upon these aspects.